test/arborist/build-ideal-tree.js

```diff
@@ -205,6 +205,15 @@ t.test('dedupe example - deduped', t => {
   return t.resolveMatchSnapshot(printIdeal(path), 'dedupe testing')
 })
 
+t.test('expose explicitRequest', async t => {
+  const path = resolve(__dirname, '../fixtures/simple')
+  const arb = new Arborist({ path })
+  const tree = await arb.buildIdealTree({ add: [ 'abbrev' ] })
+  t.ok(arb.explicitRequests, 'exposes the explicit request')
```

Comment:
Should test type and value, too.  Something like this: `t.strictSame(arb.explicitRequests, new Set(['abbrev']))`

Reply:
Totally, will add this!